server/data/abstractHmppsRestClient.ts

```diff
@@ -0,0 +1,115 @@
+import superagent, { SuperAgentRequest } from 'superagent'
+import Agent, { HttpsAgent } from 'agentkeepalive'
+
+import logger from '../../logger'
+import sanitiseError from '../sanitisedError'
+import config, { ApiConfig } from '../config'
+import TokenStore from './tokenStore'
+import { createRedisClient } from './redisClient'
+import generateOauthClientToken from '../authentication/clientCredentials'
+import { ServiceUser } from '../@types/express'
+
+export interface Request {
```

Comment:
Justr a question really - would this type not be easily confusabel with the express Request type? I think it was named GetRequest, in the legacy RestClient module

Reply:
Thank you, I realised that this does not need to be exported outside of this file, so I removed the export. It shouldnt be confused now.